.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: bug'
+assignees: mastercoms
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How you encountered the issue**
+Tell us the steps on how to recreate the bug.
```

Comment:
Make these comments `<!--like this-->`

Reply:
i dont know about this one, it is without it by default

Reply:
It's annoying to delete the default text every time.

Reply:
If you use software with that many bugs that you find deleting couple sentences annoying then it is your problem.